Swat/SwatEntry.php

```diff
@@ -1,80 +1,77 @@
 <?php
 
 /**
- * A single line text entry widget
+ * A single line text entry widget.
  *
- * @package   Swat
  * @copyright 2004-2023 silverorange
  * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
  */
 class SwatEntry extends SwatInputControl implements SwatState
 {
-    // {{{ public properties
-
     /**
-     * Entry value
+     * Entry value.
      *
      * Text content of the widget, or null.
      *
      * @var string
      */
-    public $value = null;
+    public $value;
```

Comment:
Should types where the explicit null is erased get nullable added to their type doc?